README.md

```diff
@@ -157,3 +157,30 @@ This section applies to Solidity and JavaScript projects alike.
 [soliumrc]: ./.soliumrc.json
 [travisci]: https://travis-ci.org/
 [webpack]: https://webpack.js.org/
+
+## Release Process
+
+* First version to be released is alpha version, followed by beta version.
```

Comment:
Alpha release should be done when the feature we are completing is partially complete or incomplete.
Beta release should be done when the feature is complete and all the development is done but there are possibilities that it could contain some bugs.

Reply:
So we can only have a release when we can rule out the possibility of having a bug? 🧐

Reply:
@gulshanvasnani: perhaps it would be useful to distinguish between a tag and a release (tag without pre-release identifiers like `alpha.alphaversion` + release notes)?

So, perhaps the following changes:
- > If alpha/beta version is to be ~released~ tagged, then the tag name should include alpha/beta as per the naming convention mentioned.
- > ~Release~ For commits which have been reviewed by external auditor, a rcversion for it should be ~released~ tagged

@schemar: I expect we only release when we have high confidence there are no bugs (that are critical and that we intend to fix before releasing), but we tag as appropriate to reflect the contents and/or understanding of the commit per our guidelines. All releases include tags but not all tags are part of releases.

Thoughts?